client/src/redux/show/showReducer.js

```diff
@@ -55,7 +62,7 @@ export function showReducer(state = initialState, { type, payload }) {
       const searchParams = payload.params;
       return {
         ...state,
-        data: { ...state.data, ...searchData },
+        data: searchData,
```

Comment:
@jberns fyi - I'm changing the reducer for finding shows so that it replaces the existing data instead of appending it. Otherwise, there's a bunch of extra show data if you navigate from the calendar to Show Builder. Lmk if this works for you - if not, we can modify things so there's a way to find shows with appending to the existing shows and a separate way to do it with replacing them.

Reply:
I'm fine with this, it just means each refresh will be fresh data. We can also update the calendar to only grab the minimum of a weeks worth.

From a refresh side I don't think it would be as nice with the calendar as each week would have to be refetched at the time of navigation.

Alternatively can we run the SHOW_CLEAR when the calendar unmounts, or the show builder mounts and then you have fresh data in the show builder?

Reply:
@jberns good idea on SHOW_CLEAR, I've got that set up now and undone the change to the show reducer. So we can just keep the show calendar code as is!